plugins/fs/guest-js/index.ts

```diff
@@ -1093,42 +1093,76 @@ interface WatchOptions {
 interface DebouncedWatchOptions extends WatchOptions {
   /** Debounce delay */
   delayMs?: number;
+  /** Keep track of the file system IDs of all files */
+  trackFileIds?: boolean;
 }
 
 /**
  * @since 2.0.0
  */
-type RawEvent = {
-  type: RawEventKind;
+type Event = {
```

Comment:
Let's rename this to `WatchEvent` and similar do the same to the other types so it conveys that these events are related to the watcher.